content/guides/dashboard/github-integration.md

```diff
@@ -88,6 +107,78 @@ you choose **All repositories**.
 
 3. Click the **Install** button to complete the installation.
 
+### Cypress GitHub Enterprise app installation process
+
+For GitHub Enterprise the installation is a little more involved, you need to
+create a new GitHub App, copy various settings and credentials from the new app,
+paste them into the Dashboard and go through the GitHub app activation process.
+
```

Comment:
Run on sentence. Suggest...
To install GitHub Enterprise Integration you need to; create a new GitHub App, copy the required settings and credentials from the new app to the Cypress configuration form, and complete the GitHub app activation process.